src/utils/hash.ts

```diff
@@ -26,6 +26,10 @@ export async function hashValue(value: string, dataType: string, jwt?: string) {
 
   if (!hashValueAlreadyExists) {
     await encryptValue(value, hashedValue, dataType, jwt);
+    // set the value so that we don't try to re-encrypt it
```

Comment:
what happens if the encryptValue call fails?

Reply:
skip and log it